pkg/access/group_target.go

```diff
@@ -44,10 +45,10 @@ type Grant struct {
 	Subject string                               `json:"subject" dynamodbav:"subject"`
 	Status  types.RequestAccessGroupTargetStatus `json:"status" dynamodbav:"status"`
 	//the time which the grant starts
-	Start time.Time `json:"start" dynamodbav:"start"`
+	Start iso8601.Time `json:"start" dynamodbav:"start"`
 	//the time the grant is scheduled to end
-	End          time.Time `json:"end" dynamodbav:"end"`
-	Instructions *string   `json:"instructions" dynamodbav:"instructions"`
+	End          iso8601.Time `json:"end" dynamodbav:"end"`
+	Instructions *string      `json:"instructions" dynamodbav:"instructions"`
```

Comment:
Access instructions were intended to be saved as a seperate type in dynamo db, there is already an instructions data model, the idea was that the FE would fetch instructions seperately to the request itself